pkg/terminal/command.go

```diff
@@ -650,7 +660,9 @@ func goroutines(t *Term, ctx callContext, argstr string) error {
 			case "-s":
 				fgl = fglStart
 			case "-t":
-				bPrintStack = true
+				flags |= printGoroutinesStack
+			case "-l":
```

Comment:
This new flag needs to be documented: https://github.com/go-delve/delve/blob/2359e6b2ff949a536e636a1a5c8f7d30f630fe0a/pkg/terminal/command.go#L192.